acslib/ccure/crud.py

```diff
@@ -228,3 +234,119 @@ def delete(self, record_id: int) -> ACSRequestResponse:
                 headers=self.connection.base_headers
             )
         )
+
+
+class CcureClearanceItem(CcureACS):
+    def __init__(self, connection: Optional[CcureConnection] = None):
+        super().__init__(connection)
+        self.default_search_filter = ClearanceItemFilter()
+        self.item_types = ClearanceItemTypes
+        self.type_longifier = {
```

Comment:
how about `type_map`? :D

Reply:
Also you can add a method to the `ClearanceItemType` that gives the long version

Reply:
something like this on the ClearanceItemType
```python
    @property
    def complete(self):
        if self == self.DOOR:
            return "SoftwareHouse.NextGen.Common.SecurityObjects.Door"
        if self == self.ELEVATOR:
            return "SoftwareHouse.NextGen.Common.SecurityObjects.Elevator"
```
This makes it reusable everywhere someone might use the type.

Reply:
I've just changed it to `type_map` for now, but I'm curious about ways to handle this better. I haven't been able to get anything like what you're describing above to work, maybe you can show me what I'm missing.

Reply:
Ah, I see what you did. I was getting stuck on using the string `"door"` in the `search` call instead of `cit.DOOR`. 
Thanks for the clarification, I like this